modules/users/server/config/strategies/jwt.js

```diff
@@ -0,0 +1,32 @@
+﻿'use strict';
+
+/**
+ * Module dependencies
+ */
+var passport = require('passport'),
+  JwtStrategy = require('passport-jwt').Strategy,
+  ExtractJwt = require('passport-jwt').ExtractJwt,
+  User = require('mongoose').model('User');
+
+module.exports = function (config) {
+  var opts = {
+    jwtFromRequest: ExtractJwt.versionOneCompatibility({ tokenQueryParameterName: 'auth_token' }),
```

Comment:
I was digging up more info about this:

- http://stackoverflow.com/a/27618749/1984644
- https://github.com/facundoolano/socketio-auth
- https://github.com/socketio/engine.io-client/pull/379